SFML.Module.props

```diff
@@ -17,7 +17,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="CSFML" Version="2.6.1" />
+    <PackageReference Include="CSFML" Version="[2.6.1]" />
```

Comment:
Should this be limited up to the patch version?

In SFML.Net 2.5.* it was useful to update CSFML until we provided a new SFML.Net version